code/_onclick/other_mobs.dm

```diff
@@ -115,7 +117,13 @@
 /*
 	Monkeys
 */
-/mob/living/carbon/monkey/UnarmedAttack(atom/A)
+/mob/living/carbon/monkey/UnarmedAttack(atom/A, proximity)
+	var/override = 0
+	for(var/datum/mutation/HM as() in dna.mutations)
+		override += HM.on_attack_hand(A, proximity)
+	if(override)
+		return
```

Comment:
What is the point of override?
Returning at the end of a proc will do nothing, so this is essentially the same as
```
/mob/living/carbon/monkey/UnarmedAttack(atom/A, proximity)
	for(var/datum/mutation/HM as() in dna.mutations)
		HM.on_attack_hand(A, proximity)
```

Reply:
Override stops attack_paw from being called on line 127